src/gui/widgets/Fader.cpp

```diff
@@ -73,7 +73,8 @@ SimpleTextFloat* Fader::s_textFloat = nullptr;
 Fader::Fader(FloatModel* model, const QString& name, QWidget* parent, bool modelIsLinear) :
 	QWidget(parent),
 	FloatModelView(model, this),
-	m_modelIsLinear(modelIsLinear)
+	m_modelIsLinear(modelIsLinear),
+	m_knobSize(m_knob.size() / m_knob.devicePixelRatio())
```

Comment:
`warning: field 'm_modelIsLinear' will be initialized after field 'm_knobSize' [-Wreorder-ctor]`

```suggestion
	m_knobSize(m_knob.size() / m_knob.devicePixelRatio()),
	m_modelIsLinear(modelIsLinear)
```